roles/wazuh/ansible-wazuh-agent/tasks/Debian.yml

```diff
@@ -25,6 +25,7 @@
 - name: Debian/Ubuntu | Installing Wazuh repository key
   apt_key:
     url: "{{ wazuh_agent_config.repo.gpg }}"
+    id: "{{ wazuhrepo.key_id }}"
```

Comment:
Please initialize the variable in the defaults installation for Manager and Agent.